src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -503,6 +503,11 @@
         <xsd:attribute name="local-pk" type="xsd:string" />
         <xsd:attribute name="passphrase" type="xsd:string" />
         <xsd:attribute name="ciphers" type="xsd:string" />
+        <xsd:attribute name="buffer">
+            <xsd:simpleType>
+                <xsd:union memberTypes="xsd:string xsd:boolean" />
```

Comment:
I think `type="xsd:string"` would cover all cases

Reply:
Simplified the configuration. Looks like it's working